commit b7c9bb183b799b10c09ec32e98d1843546ea4324
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Wed Jun 17 05:32:37 2020 -0700

    x86: Correct bit_cpu_CLFLUSHOPT [BZ #26128]
    
    bit_cpu_CLFLUSHOPT should be (1u << 23), not (1u << 22).

diff --git a/sysdeps/x86/cpu-features.h b/sysdeps/x86/cpu-features.h
index 388172a1c07bf979..7abfa046c4b8f6ac 100644
--- a/sysdeps/x86/cpu-features.h
+++ b/sysdeps/x86/cpu-features.h
@@ -465,7 +465,7 @@ extern const struct cpu_features *__get_cpu_features (void)
 #define bit_cpu_ADX		(1u << 19)
 #define bit_cpu_SMAP		(1u << 20)
 #define bit_cpu_AVX512_IFMA	(1u << 21)
-#define bit_cpu_CLFLUSHOPT	(1u << 22)
+#define bit_cpu_CLFLUSHOPT	(1u << 23)
 #define bit_cpu_CLWB		(1u << 24)
 #define bit_cpu_TRACE		(1u << 25)
 #define bit_cpu_AVX512PF	(1u << 26)
